core/src/i18n/en.yaml

```
@@ -823,7 +837,11 @@ functions:
     unhealthy: Unhealthy
   title: Functions
   tooltips:
+    building: Waiting for the image to be build.
+    deploying: Deploying function.
```

```suggestion
    deploying: Deploying the Function.
```
We [capitalize `Function`](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) as a proper name.